env.js

```diff
@@ -24,7 +24,6 @@ const PLUGINS_MODULE = process.env.PLUGINS_MODULE;
 const STATUS_URL = process.env.STATUS_URL;
 
 // Configure Google Analytics
-const ENABLE_GA = process.env.ENABLE_GA || false;
 const GA_TRACKING_ID = process.env.GA_TRACKING_ID || 'G-0QW4DJWJ20';
```

Comment:
Looks like it was added in [PR #244](https://github.com/flyteorg/flyteconsole/pull/244/files) and wasn't touched at all at commit [#251](https://github.com/flyteorg/flyteconsole/pull/251)

After you remove it - would we still rely on ENABLE_GA environment variable to turn on/off analytics? 
If not - the ReadMe files need to be updated too.